Client/Frontend/Browser/BrowserViewController/BrowserViewController+KeyCommands.swift

```diff
@@ -47,8 +47,12 @@ extension BrowserViewController {
 
     @objc
     func addBookmarkKeyCommand() {
-        if let tab = tabManager.selectedTab, homepageViewController?.view.alpha == 0 {
-            guard let url = tab.canonicalURL?.displayURL else { return }
+        guard let tab = tabManager.selectedTab,
+              let url = tab.canonicalURL?.displayURL else { return }
+
+        if AppConstants.useCoordinators, contentContainer.hasHomepage {
```

Comment:
Unless I'm misreading it I think most of these checks are backwards. Home page alpha == 0 would be the equivalent of the contentContainer not having a homepage.

Reply:
Yeah you're right, I inverted that 😣